junit-jupiter-engine/src/test/kotlin/org/junit/jupiter/api/ContextScope.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+package org.junit.jupiter.api
+
+interface ContextScope {
```

Comment:
Related to my previous comment, but this interface doesn't seem to be useful. See previous comment for explanation.